frontend/packages/console-dynamic-plugin-sdk/src/app/core/reducers/coreSelectors.ts

```diff
@@ -1,7 +1,8 @@
-import { ImpersonateKind, SDKStoreState, UserKind } from '../../redux-types';
+import { UserInfo } from '../../../extensions';
+import { ImpersonateKind, SDKStoreState } from '../../redux-types';
 
 type GetImpersonate = (state: SDKStoreState) => ImpersonateKind;
-type GetUser = (state: SDKStoreState) => UserKind;
+type GetUser = (state: SDKStoreState) => UserInfo;
```

Comment:
I was worried that this might be an API breaking change in our plugin SDK, but looking closer, I don't believe we expose this in a public API. Probably want another set of eyes (maybe @TheRealJon) to confirm.